waspc/cli/Wasp/Cli/Command.hs

```diff
@@ -22,4 +22,4 @@ runCommand cmd = do
 
 -- TODO: What if we want to recognize errors in order to handle them?
 --   Should we add _commandErrorType? Should CommandError be parametrized by it, is that even possible?
-data CommandError = CommandError {_errorMsg :: !String}
+newtype CommandError = CommandError {_errorMsg :: String}
```

Comment:
I usually use `data` instead of `newtype` when I could use `newtype` if I know that the type is likely to expand with more fields in the future.

I understand that `newtype` is faster due to not existing in runtime, but I still like to have these semantics that `data` brings. I would actually suggest we turn off this hlinter warning. What do you think?

Reply:
I prefer `newtype` where possible. As we have hlint and stan set up, we'll get reminded to put back the strictness bang should we forget when going back to `data` for a type.

Reply:
Aha, so the strictness bang gets removed because of newtype -> makes sense.

I have to admit I still prefer semantics that `data` brings over possible performance gain that `newtype` brings, so I would go for ignoring that hlint rule.